examples/f2py_string_input/Makefile

```diff
@@ -31,5 +31,5 @@ f2py-ko: ${OBJ} ${SIGNATURES}
 f2py-ok: ${F90_SRC}
 	CFLAGS="${CFLAGS}" ${F2PY} -c -m _${PY_MOD} ${F2PYFLAGS} ${F90_SRC}
 
-test: f2py-ko
+test: f2py-ok
```

Comment:
The goal of this test is specifically to check that what the f2py-ko rule does is working. The f2py-ok rule is there for the debugging process, proving that this is an issue specific to using signatures.